# <center>Biology Final Review</center>

## Phylogeny, Population Genetics and Population Growth

Refer to Study Guides

## Macromolecular Self-Assembly

### Lipid Bilayers

- Biological Compartments
  - Intermolecular regions
  - A membrane limited space within which molecules are free to diffuse
  - selectively permeable to membrane
- Amphipathic molecules
  - Hydrophobic + Hydrophilic regions
  - Component of biological membranes
  - Distinct, separate, regions with different solubility properties
- Plasma membrane
  - Lipids + Proteins
  - Surrounds a cell
  - Separates the internal cell and the external environment
  - Regulates passage of molecules and ions into/out of the cell
- Features of Membrane Lipids (Phospholipids)
  - Polar + non-polar region (Amphipathic)
    - Polar head groups: interacts strongly with water
    - Acyl tails: do not interact with water
    - Creates a water-free zone in the center of the bilayer
  - Hydrocarbon chain
  - Bonded with oxygen
- Phospholipids form either micelles or bilayers when suspended in water
- Molecular Self-Assembly
  - Molecules/Complexes of molecules form a particular and characteristic 3D conformation or structure
  - Result of
    - Interaction with surrounding water molecules
    - Intermolecular interaction
    - Intramolecular interaction
    - Hydrophobic + Hydrophilic interactions between and within molecules
  - Extends from single molecules to complexes to organelles
- Hydrophobic association
  - Two stages
    - Entropy driven clumping of hydrophobic groups
    - Formation of L-vdW interactions between adjacent hydrophobic groups
  - Hydrogen bonds in adjacent water molecules are reoriented tangentially to the surface
  - Inner layers of water are highly constrained
  - Reduced entropy: more structured than "free" water; reduced hydrogen bonding potential
  - Hydrophobic groups "squeeze" the water out of the region, increasing entropy and hydrogen bonds
  - $\Delta G = \Delta H - T(\Delta S_{water} + \Delta S_{phospholipids})$
- Hydrophilic interactions
  - To water; to polar molecular groups
  - Ionic interaction

### Properties of the Membrane

- Which forms more readily?
  - Micelles - Maximizes the area of hydrophilic interactions (large free energy change associated)
    - Intermediate end for detergents (detergent destroys bilayers and breaks into micelles)
  - Bilayer (energy barrier to overcome from hydrophobic tail reentering water)
    - Driving force to form bilayers
      - Entropy
      - L-vdW interactions
      - Brownian motion
- General properties of membrane phospholipid bilayer
  - Selective permeability
    - Small, non-polar molecules move across quickly
    - Charged, large molecules cross slowly
    - Permeability vs. Hydrophobicity
      - High hydrophobicity, high permeability
      - High hydrophilicity, low permeability
    - Diffusion rates
      - Small hydrophobic molecules > Small uncharged polar molecules > Larger uncharged polar molecules > Ions
    - Osmosis
      - Solutions of different concentrations are separated by a membrane that is permeable to water but not to the solutes
      - Water moves to solution with higher solute concentration
      - Hypertonic: $[in] < [out]$; Isotonic: $[in] = [out]$; Hypotonic: $[in] > [out]$
      - Prevention of cell explosion
        - Maintain isotonic environment
        - Build a cell wall
        - "Secrete" water by secreting ions
        - Better membrane less permeable to water
  - Self-healing/sealing
    - Small hole $\to$ Membrane will close it
    - Torn into fragments $\to$ Vesicles
  - 2D fluidity
    - Weak bonds from L-vdW interactions and hydrogen bonding
    - Individual molecules can move within the bilayer
    - Free to diffuse within the plane of the membrane
    - Hard for phospholipid leaflets to flip
      - Hydrophilic groups breaking hydrogen bonds - high energy barrier
      - Hydrophobic groups prevent charged molecules from passing through
    - Fluorescence recovery after photobleaching (FRAP)
    - Acclimation to temperature change
      - Decreasing temperature, decreased fluidity; Increasing temperature, increased fluidity
    - Fatty acid composition
      - Saturated acid: Lack double bonds, favours tight packing in membrane bilayers
      - Unsaturated acid: has double bonds, reducing the tightness of packing in membrane bilayers
      - Long hydrophobic chain: Increases L-vdW interactions; short hydrophobic chain: decreases L-vdW interactions
  - Protein container
  - Separate compartments

### Proteins and Amino Acids

- Amino Acid Structure
  - Amino group + Carboxyl group + $\alpha$-Carbon + Side chain
  - Hydrophilic and hydrophobic amino acids
- Protein Function
  - Antibodies and complement proteins: Defense - destruction of foreign bacteria and viruses
  - Contractile and motor proteins - Movement
  - Enzymes - Catalyze chemical reactions
  - Peptide hormones - Act as signals to coordinate activities of cells
  - Receptor proteins - Receive signals from outside the cell and initiate response
  - Structural proteins - Provide support for cells and tissues
  - Transport proteins - Move ions or molecule across membranes
-  Proteins = Polymers of AA
  - Polypeptide bonds: consistent internal polarity (N to C)
- Structure of Proteins
  - Primary Structure: Amino acid sequence (Start with a N-terminus, end with a C-terminus)
  - Secondary Structure: Formation of hydrogen bonds between atoms of polypeptide backbone with other AA ($\alpha$-helix and $\beta$-sheet)
    - Proteins typically contain regions that form both structures
    - $\alpha$-helix: Each carbonyl group forms a hydrogen bond with an amide group 4 residues away
    - $\beta$-sheet: Adjacent strands can run in parallel or antiparallel
    - What determines which shape to form?
      - Different side chains
      - Side-chain interactions
      - Side-chain - water interactions
  - Tertiary Structure: 3D shape of a protein
    - Stabilized by interactions of side chains with other side chains or atoms of the polypeptide backbone
      - Hydrophobic/hydrophilic/ionic association
      - Disulfide bonding (formed in endoplasmic reticulum by enzyme)
    - X-ray diffraction
    - In globular proteins
      - Hydrophilic groups located on the surface
      - Hydrophobic groups located inside
  - Quaternary Structure: Interactions of protein subunits
    - More than one subunit
    - Stabilized by all of the forces that stabilize tertiary structure

- Self-Assembly of a Protein
  - Random thermal movements of protein, of water molecules, and by the interaction of different chemical groups along with backbone of protein, with water to achieve their normal minimum energy configuration
  - Configuration is through trial & error
  - Chaperone: chaperon fully synthesized proteins, ensure proper native folding
- Amino acid substitution in proteins (mutations)
  - Hydrophobicity/hydrophilicity of amino acids and the change in conformation of proteins
  - Assume substitution of an amino acid of a particular type by a different amino acid of the same class will have relatively little effect on the structure of the molecule (Hydrophobic replaces hydrophobic)
- Protein denaturation
  - Denaturation agents: heat, molecules that interfere with hydrogen bonds (urea)
  - Often irreversible
  - Can renature if careful
  - Urea is a strong hydrogen bond competitor, so it disrupts hydrogen bond forming intramolecularly
  - Renaturation
    - Place a dialysis bag (with urea) in distilled water for hydrogen bonds to reform

### Membrane Proteins

- Roles
  - Transporter of molecules and ions
  - Enzymes (electron carriers, ATP synthase)
  - Receptor and Signal transducers
  - Anchors for other proteins (cytoskeleton)
- Types
  - Integral membrane proteins: contain one or more membrane crossing domains (glycophorin)
    - Embedded in the membrane
    - Each membrane crossing domain contains regions of hydrophobic amino acid residues that traverses the hydrophobic core of the membrane
    - Sequences of amino acids with primarily hydrophobic side chains
    - Hydrophobic side chains in hydrophobic regions + Hydrophilic side chains in hydrophilic regions
    - Charged on the outer surface of the bilayer
  - Peripheral membrane proteins: attached to the outside of the membrane by interaction with integral membrane proteins
- Hydropathy plots
  - Shows free energy change associated with immersion sequential groups of 3 amino acids in water (start at the N end)
    - Positive free energy change (hydrophobic)
    - Negative free energy change (hydrophilic)

### Processing and Targeting of Proteins

- Proteins must be targeted to their proper destinations
  - Cytosolic proteins
    - Folded, covalent modification, cleavage
  - Proteins of the endomembrane system (ER/Golgi system)
    - Folded, disulfide bond formation, Glycosylation, cleavage
- Targeting (Signal Sequence + Receptor)
  - Signals encoded within proteins
  - Signals direct the protein to a specific organelle
  - Signals must be present for protein to leave the cytosol compartment
  - General Principle
    - Regulatory signals required for the processing or function of macromolecules are generally encoded within the macromolecules themselves
  - Options
    - No targeting signal - remain in cytosol
    - N-terminal signal - targeting to ER or organelles other than nucleus
      - Signal sequence cleaved off
    - Internal targeting signal - targeting to nucleus
      - Signal sequence not cleaved off
  - Requirements
    - Specific signal sequence in the transported protein
    - Specific protein receptor that recognizes that signal sequence
- Targeting to endomembrane system
  - Co-translational insertion into endoplasmic reticulum
  - Proteins then proceed by vesicle transport to Golgi Apparatus and then on to the plasma membrane or other destination
- Rough ER
  - Ribosomes on ER translate protein destined for the ER or membrane proteins
  - Translation of all proteins, regardless of destination, begins in the cytosol
- Co-translational Insertion
  - Signal Recognition Particle binds to a signal sequence and halts translation
  - SRP binds to the SRP receptor on the ER membrane
  - SRP receptor brings the ribosome to a transmembrane channel, SRP dissociates; protein synthesis resumes, and the growing polypeptide chain is threaded through the channel
  - Protein ends up in the lumen of ER for further action
  - Once enter ER, no returning to cytosol
- Targeting of proteins to membranes
  - Membrane proteins are co-translationally inserted
  - Arrangement of the protein in the membrane is controlled by amino acid sequences in the protein (Start and Stop transfer signals)
  - Stop transfer sequence
    - Stops transfer of protein
    - Contain a membrane crossing domain
    - May have same sequence as internal start sequence
    - **IMPORTANT**: order in which these regions are encountered

## Regulation of Genetic Expressions

### Lac Operon - Regulation of transcription in bacteria

- Trans-acting regulatory factor (external factor that binds to a regulatory DNA sequence) + cis-acting regulatory sequence = gene (in)activity
- Regulatory Region
  - Activator binding site - When protein binds transcription occurs much more frequently
  - Promoter - RNA polymerase binding site
  - Repressor site - When protein binds transcription is blocked
  - Positive regulation
    - Transcription only occurs when an activator protein binds to a site near the promoter and then RNA polymerase can bind to the promoter
  - Negative regulation
    - Transcription can occur when only RNA polymerase binds to the promoter
    - If repressor protein binds to the DNA and inhibits RNA polymerase binding to the promoter, no transcription occurs
- Compared to eukaryotes, prokaryotes
  - DNA is not packaged in chromosomes
  - mRNA is not processed
  - No nuclear membrane that separates transcription and translation
- Lactose catabolism in *E. coli*
  - Can use lactose as sole energy and carbon source
  - $\beta$-galactosidase required to break down lactose
  - Cells grown on glucose cannot break down lactose, have very little $\beta$-galactosidase
  - In the absence of glucose, $\beta$-galactosidase enzyme activity **induced** by lactose (control of expression by environmental factor)
- Constitutive - always being active/produced; Facultative - active/produced only under specific conditions
- Lac Operon
  - *lacI*: structural gene for the repressor protein
  - CRP-cAMP binding site
  - Promoter
  - *lacO*: the binding site for the *lac* repressor protein
    - In the absence of lactose, repressor protein translated by *lacI* binds to *lacO* and prevents transcription
    - In the presence of lactose, repressor protein is induced by lactose (allosteric regulation), unable to bind to *lacO*, promoter recruits RNA polymerase complex and transcription of **polycistronic** mRNA occurs
    - Lac repressor binding sites at each side of *lacO* sequence
    - Each site has two binding sequences in opposite orientation
    - Repressor tetramer complex has two dimers, one of which binds to each binding site
  - *lacZ*: coding sequence for $\beta$-galactosidase
  - *lacY*: coding sequence for galactoside permease
- General principle
  - Binding of a ligand to a protein changes the conformation of the protein
  - To get *lac* activity
    - Lactose present to remove *lac* repressor
    - Glucose absent or low to allow activation of CRP activator of Lac Operon by cAMP
- Effect of glucose on Lac Operon
  - Cells growing in presence of glucose use glucose
  - If switched to lactose, will use lactose $\to$ induction of $\beta$-galactosidase activity
  - When both are present, glucose is preferred
  - $Lactose \xrightarrow{\beta-galactosidase}Galactose + Glucose$
  - Glucose inhibits $\beta$-galactosidase synthesis
  - CRP protein (binds to CRP binding site - an activator) activated by cAMP
    - $ATP \xrightarrow{Adenyl \space cyclase} cAMP+pyrophosphate$
    - High glucose, low adenyl cyclase activity, low cAMP production, no activation of Lac Operon transcription via activated CRP
- Summary
  - Multiple switches for genetic regulating expression (positive and negative control)
  - Transcription activity is regulated by DNA binding proteins
  - Signals control gene expression by modulating transcription factor activity
  - Transcription factor activity is regulated by ligand binding
  - Affinity of enzymes for substrates can be compared to affinity of transcription factors for DNA binding sites
  - Mutations can define genes and gene regulatory regions

### Eukaryotic Regulatory Gene Expression

- Transcription Factor Binding
  - Gel shift analysis to determine which DNA fragments bind to a particular protein
    - Binding of protein reduces DNA mobility in an electrophoretic gel
  - Footprint analysis to determine the location of protein binding sites
    - Tight binding of a protein to DNA prevents nuclease (DNase I) from binding to the DNA and from cutting the molecule
  - Gel shift analysis for target competition experiments to assess affinity of TF / DNA binding
    - Binding of protein to DNA produces a "gel shift" - a change in the position of the DNA band that can be used to estimate the relative amount of bound DNA
  - Quantitative assessment of TF binding
    - $Ligand+Protein \leftrightarrow PL complex$
    - $K_a = \frac{[PL]}{[P][L]}$
    - Plot fraction of protein with bound ligand vs. ligand/DNA concentration
      - $K_d$ concentration of substrate at which $50%$ of the protein molecules are complexed with ligand
      - $Fraction \space bound \space (\%saturation)=\frac{[ligand]}{K_d + [ligand]}$
      - Lowest affinity, largest $K_d$

- Eukaryotes
  - Similar model to prokaryotic regulation
  - Regulatory Regions
    - Promoter: Has sequence common to most genes
      - Most common sequence: TATA box
      - All eukaryotic promoters are bound by the TATA-binding protein
    - Promoter proximal element: Has sequence that is unique to this gene
      - Coordinately regulated genes often have similar sequenced PPE
      - Binding sites for regulatory transcription factors
      - Located just upstream of the promoter
  - Transcriptional Regulation
    - Specific TF bind to
      - PPE
      - Enhancer regions that are at a distance from the promoter
    - All eukaryotes have enhancers
      - May locate upstream, downstream, or within the transcription unit; in introns, in untranscribed 5' or 3' sequences
    - Enhancer - Increasing rate of transcription
      - Regulatory proteins bind to these are activators/co-activators
    - Silencer - Decreasing rate of transcription
      - Regulatory proteins bind to these are repressors/co-repressors
    - Enhancers work through DNA bending to bring the associated TFs into contact with the proteins of the basal transcription complex
  - Different cell types express different genes because they have different histone modifications and contain different regulatory proteins
    - Certain proteins produced only in certain types of cells
  - Coordinated control of gene expression
    - A single TF may interact with many target genes
    - Action can be positive or negative
    - TF act in Boolean logic to control transcription
    - A single gene can be acted upon by many TFs
    - Britten and Davidson model

### Making a Fly

- Preformation
  - Important for establishing axial coordinate system of the egg
  - Maternal mRNAs are placed at the poles of the oocyte by the cells of the mother's ovary, defining the anterior-posterior axis of the embryo
  - *bicoid* is localized at the anterior pole of the oocyte
- Events of early development in the fly
  - Cleavage
    - Egg is one big cell
    - After fertilization, nuclei divide without forming cell membranes
    - One cell, many nuclei
    - 13 rounds of DNA replication followed by nuclear division happen in one single giant cell
    - Nuclei arranged around the periphery of the cell $\to$ Syncytial blastoderm stage
  - Cellularization
    - Cycle 14 - synchronous formation of cells all around the periphery of the egg $\to$ Cellular blastoderm stage
    - Followed by gastrulation $\to$ internal cell mass
  - *bicoid* mRNAs are translated and give rise to a gradient of *bcd* protein (diffuses away from the source and establishes a gradient throughout the embryo)
- Loss of *bcd* protein: no anterior development; Loss of *nos* protein: no posterior development
- Morphogens
  - A substance that supplies positional information that governs the formation of a characteristic structure in a developing embryo
    - *bcd* protein: anterior morphogen; *nos* protein: posterior morphogen
    - *bcd* protein gradient is formed after fertilization, mRNA is still localized
    - mRNA is not free to diffuse, protein is free to diffuse
- Modeling the *bcd* gradient
  - SDD model: synthesis, diffusion, destruction
  - Protein produced by mRNA at the anterior continually
  - Diffuses outward
  - Protein has a characteristic life time in which a fixed fraction of the protein is lost
- Big Ideas
  - 1
    - Gene expression in development is hierarchical
    - Genes that are active early in development produce proteins that regulate activities of genes that act later
    - Maternal gradients $\to$ Gap genes $\to$ Pair-rule genes $\to$ Segment polarity genes $\to$ Homeotic genes ...
  - 2
    - Different genes are expressed in different regions of the embryo
    - Gradients are established in different regions
    - These proteins are morphogens
    - Different target genes are activated or repressed at **different** concentrations of TF protein
    - Translation of *hb* inhibited by *nos*; Translation of *cd* is inhibited by *bcd*; Transcription of *hb* is activated by *bcd*...
- Gap genes
  - Different gap genes active in different regions based on the concentrations of *bcd*, *hb* and other TFs
  - *hb* boundary sharpening
    - Posterior boundary of the *hb* expression domain becomes progressively more sharply defined
    - Consequence of interaction of *bcd* and *hb* proteins with the *hb* promoter
    - Positive feedback: *hb* protein binds to *hb* promoter to activate *hb* expression
  - Organization of *hb* promoter region
    - Multiple *hb* and *bcd* binding sites
    - Binding of multiple activating TFs (increased frequency of transcription)
    - Cooperative binding - binding of a TF to a strong site increases probability of binding of the same TF to an adjacent weak site
  - Require high concentration (low affinity), require low concentration (high affinity) for activation
  - Kruppel promoter organization - presence of multiple *hb* and *bcd* binding sites
    - Cooperative binding from *bcd* and *hb*
    - *bcd* is an activator
    - *hb* is a repressor
  - Gap gene expression divides the embryo into zones
    - Gap gene products are TFs that are also morphogens (shorter distance)
    - Regulate expression of segmentation genes
    - Gap genes inhibit each other
  - Competitive binding
    - Same binding site is not large enough for more than one TF
    - Either A or B
      - Dependent on concentration and affinity
- Pair rule genes (regulated by gap gene products)
  - Even-skipped - Expressed in odd numbered segments
    - Expressed in 7 bands each corresponding to an odd numbered segment
    - Each stripe controlled by a particular enhancer element
    - Two sets of enhancers (embryonic enhancers and other enhancers that act later after cellularization)
    - Stripe 2 (*eve-2*)
      - Inhibited by *Gt* and *Kr* proteins
      - Activated by *bcd* and *hb* proteins
      - Competitive binding of TFs (needs higher affinity in farther gradients)
  - Odd-skipped - Expressed in even numbered segments
- Homeotic Genes (Segment polarity genes set up cues to example accurate expression of homeotic genes)
  - Specify and identity of a body part or segment during embryonic development
  - *Antennapedia* gene (antenna development)
  - *Bithorax* gene (wing development)
  - *Hox* gene
    - AA sequence of the DNA binding domains (homeodomains) of *Hox* gene products are very similar from one organism to the next
    - Fly: segments
    - Mice: spinal cords
    - Plants: buds, leaves, flowers
- Summary
  - Initial positional cues come from TF gradients
  - With passage of time domains in the embryo become defined with increasing precision
  - Results in a pattern of segments that differentiate in response to their position in the embryo based on the local pattern of gene activity
  - This pattern leads to activation of appropriate homeotic genes that form the definitive positional information system for animals.
  - Position-specific TFs

## Biological Energy Transformation

### Enzymes and Transport Proteins

- Enzyme
  - Accelerates the reaction by reducing activation energy
  - Change in free energy is the same
  - Fraction of molecules reacting increases if activation energy decreases, increasing the rate of reaction
  - Binds specifically to reactants
  - Catalyzes the reaction in both directions
  - Does not affect equilibrium position
  - How does it work
    - Has high affinity for reactants, binds tightly to them
    - Increase local concentration of reactants by bringing them to close proximity
    - Increases probability of bond breakage and reaction
    - Products have lower affinity for enzyme
  - If protein is unfolded, no enzymatic activity; only has activity when properly folded
- Enzyme-Substrate Complex
  - Non-enzymatic: $S \leftrightarrow P$
  - Enzymatic: $E + S \leftrightarrow ES \leftrightarrow EP \leftrightarrow E + P$
- Enzyme Action
  - Initiation: Reactants bind to the active site in a specific orientation
  - Transition state facilitation: Interactions between enzyme and substrate lower the $E_{activation}$ required
  - Termination: Products have lower affinity are released, enzyme unchanged
- Energy change during enzyme reactions
  - Sub reactions
    - Formation of E-S complex
    - Main reaction - Catalysis (Rate-Limiting Step)
    - Release of products
- Regulation of enzyme function
  - Co-factors: Ions, co-enzymes
  - Inhibition
    - Competitive (Inhibitor and substrate compete for the same site)
    - Non-competitive inhibition or allosteric regulation
  - Consider Threonine dehydratase
    - Activated allosterically by threonine (reactant of the pathway)
    - Inhibited allosterically by isoleucine (product of the pathway)
- Measuring binding of ligands
  - $K_d$: half-saturation concentration
  - Binding affinity of the protein for the ligand is $\frac{1}{K_d}$, affinity high, $K_d$ is low
  - Michaelis-Menten Kinetics
- Catalyzing endergonic reactions ($\Delta G$ is positive)
  - Main energy input: ATP (bonds linking the 3 phosphates has very high potential energy)
  - Hydrolysis of ATP $\to$ energy to drive endergonic reactions
- Transport Protein
  - Simple diffusion
    - Molecules move down their concentration gradient
  - Facilitated diffusion (Passive transport)
    - Solutes interact with a transmembrane protein
    - Diffuses down its concentration gradient
    - No local application of energy is required
    - Process
      - Protein binds to substrate with moderate to high affinity
      - Configuration of protein is metastable, stochastic movement of protein exposes binding site to one side of the membrane or the other
      - Finite probability of the ligand being "spontaneously" released from the binding site
      - Ligand will flow through the protein, down the ligand concentration gradient
    - Carrier mediated facilitated diffusion - rate of transport
      - Substrate concentration
      - Avidity of substrate binding
      - Max throughput of the carrier
  - Active transport
    - Change the conformation of the protein and move the solute across the membrane against its concentration gradient to a higher free energy state (hydrolysis of ATP)
    - Moves $Na^+$ out of the cell and $K^+$ into the cell, building up an electrochemical gradient across the membrane (ATPase pump)
    - Cellular electrochemical gradient
      - Transmembrane voltage is $50mV$
      - Hydrophobic core of membrane is $3nm$ thick
      - Voltage gradient is $167kV/cm$
      - Concentration difference in ions
      - Electrical charge difference
      - $\Delta G = RT \ln(\frac{[in]}{[out]})+ZF\Delta P$
    - Indirect active transport
      - Transport protein uses the ion electrochemical gradient from above active transport to move a different molecule across the membrane against its gradient
      - $|\Delta G_{ion}| > |\Delta G_{molecule}|$
      - Symport - driving ion and transported molecule in the same direction
      - Antiport - ion and molecule in opposite direction
    - $Na^+$ glucose transporter
      - Symport system - Drives the uptake of glucose against its concentration gradient by moving down its electrochemical gradient
      - Apical epithelium of small intestine

### Glycolysis and Fermentation

- Glycolysis (Sugar $\to$ Pyruvate)
  - Energy Input and Payout phases
    - Energy input: Glucose $\xrightarrow{2ATP}$ Fructose 1,6 bisphosphate +  2ADP
    - Cleavage: F 1,6 B $\to$ 2 Glyceraldehyde 3 phosphate
    - Energy Payout: G3P $\to$ pyruvate + 2ATP + NADH
    - Net yield: 2 ATP, 2 NADH, 2 Pyruvate (per glucose)
  - Limited energy yield
  - Subject to Feedback Regulation
  - Reduction of NAD+
  - Substrate level phosphorylation of ADP
- Regulation of glycolysis
  - High ATP level inhibits phosphofructokinase (feedback inhibition)
  - ATP is both a substrate and a regulator
  - Allosteric regulation
    - Activation: ADP, AMP
    - Inhibition: ATP
  - 2 ATP binding sites on ppfk: substrate site and regulatory site

### Fermentation (electron acceptor not present)

- Electrons are not accepted
  - Oxidize NADH by sticking the electrons back to them
- Two variants of the process
  - Lactate production
  - Ethanol production
- No net oxidation
- Energy released is a consequence of molecular rearrangement producing changes in internal energy
- Limited energy yield

### Respiration

- Export electrons out of the cell to electron acceptors
- Use high energy electrons that are in the process of being exported to do work in the form of ADP phosphorylation; use potential energy of electrons to drive phosphorylation
- NADH also carries a lot of free energy
- Synthesis of Acetyl Co-A from pyruvate
  - Pyruvate enters the mitochondrial matrix via the transporter in the inner membrane
  - Pyruvate dehydrogenase oxidizes pyruvate, releasing $CO_2$, reducing NAD+ and attaching the remaining acetyl group to CoA (an acetyl carrier)
  - Acetyl-CoA is the activated from of acetate
  - Regulation
    - Negative: High ATP, Acetyl-CoA or NADH (reduce pyruvate dehydrogenase activity by phosphorylation)
    - Positive: High NAD+, CoA or ADP (increase pyruvate dehydrogenase activity by dephosphorylation)
    - PDK (PD kinase) phosphorylates PD, inhibited by pyruvate
    - PDP (PD phosphatase) dephosphorylates PD, stimulated by PEP and AMP, inhibited by ATP, NADH and Acetyl Co-A

### Citric Acid Cycle

- Occurs in mitochondrial matrix
- Cyclical reactions
- Acetate transferred from Acetyl Co-A to citric acid to start
- Eight carboxylic acids become progressively more oxidized
- Oxidation of these substrates drives reduction of electron carriers NAD and FAD
- Acetate oxidized to 2 $CO_2$ molecules
- Acetate acceptor is regenerated
- 4 reactions in which reduced electron carriers are produced store released energy in electron carriers
- Pyruvate completely oxidized to $CO_2$
- Regulation
  - $Oxaloacetate \xrightarrow{regulated \space by \space ATP} Citrate$, feedback inhibition
  - Multiple sites of feedback inhibition from ATP and NADH
  - High ATP + NADH = inhibition; High ADP + NAD = activation
- Net yield (substrate level phosphorylation)
  - 2 ATP
- Source of initial carbon skeletons for the synthesis of many biologically important molecules via many different metabolic pathways.

### Chemiosmotic Production of ATP (Oxidative Level Phosphorylation)

- Reduced electron carriers
  - Drive the formation of proton gradient
    - Phosphorylation of ADP to ATP
- Net Yield
  - 28 ATP
- Energy in electron to drive ADP phosphorylation
  - Transfer electrons to final electron acceptor
  - Make electrons do as much work as possible to store energy by building up an electrochemical gradient
  - Use the gradient to drive phosphorylation of ADP
  - Active transport run in reverse
- Occurs in mitochondria, chloroplasts and in bacteria
  - Ions moving down a gradient generating ATP rather than ATP being used to drive ions against a gradient
- Early cells
  - Mechanisms to pump protons out of the cell, powered by ATP and electron transport
  - Electron-transport-powered pumps became efficient enough to run the ATP-driven pump in reverse
- Electrochemical gradient
  - During transfer, at each transfer, there is a decrease in free energy that drives transport of $H^+$ across the membrane
  - Electrons are eventually transferred to a terminal electron acceptor
- Electron transport complexes
  - Four multi-protein electron transport complex that comprise the respiratory electron transport chain
  - They contain both enzymes (NADH dehydrogenase and electron transporter proteins and proton pump proteins)
  - Complex I and II strips electrons from NADH and FADH$_2$
  - Cytochrome C moves electron to Complex IV where oxygen is reduced to form water
- Proton electrochemical gradient - huge potential energy level (160mV), membrane potential ~533kV/cm
- ATP synthase complex - uses potential energy stored in the electrochemical gradient to drive the phosphorylation of ADP to make ATP
  - Reversible enzyme system
  - $F_0$ unit (drives rotation of the stalk) and $F_1$ unit (rotation of stalk changes the conformation in the knob portion)
  - 3 consecutive conformations: *open, loose and tight*
  - *Tight* conformation "crushes" ADP and Pi together to form ATP
  - 3 rotations form one ATP

### Evolution of Respiration

### Light Reactions of Photosynthesis

### Dark Reactions of Photosynthesis

### Mitochondria and Chloroplasts as Bacterial Cells

### Chemoautotrophs

Refer to notes for all above five topics

